processfamily/processes.py

```diff
@@ -112,16 +112,23 @@ def cpu_count():
 
 def get_process_affinity(pid=None):
     """Gets the process_affinity cores either for the current process or the given pid. Returns a list of cores"""
-    return affinity.sched_getaffinity(pid or 0)
+    try:
+        return affinity.sched_getaffinity(pid or 0)
```

Comment:
Do we not know why these methods aren't implemented all of a sudden?

Reply:
So I rewrote the `affinity` module so that it would work on Python 2 and Python 3, however on Python 2 on linux there is no way to set the affinity as far as I can tell.

I think if they weren't implemented in the first place and it would just fail on import or maybe just do nothing silently.